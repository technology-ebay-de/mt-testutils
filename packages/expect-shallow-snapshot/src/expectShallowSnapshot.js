/* global expect */

import { shallow } from 'enzyme';
export default component => expect(shallow(component).getElement()).toMatchSnapshot();
