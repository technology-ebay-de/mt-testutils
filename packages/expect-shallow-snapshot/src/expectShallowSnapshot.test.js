import React from 'react';
import expectShallowSnapshot from './expectShallowSnapshot';

const mockComponent = <div>test</div>;

describe('My mock component', () => it('renders correctly', () => expectShallowSnapshot(mockComponent)));
