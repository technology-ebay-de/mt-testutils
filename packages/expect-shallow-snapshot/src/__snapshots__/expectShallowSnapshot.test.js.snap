// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My mock component renders correctly 1`] = `
<div>
  test
</div>
`;
